--- a/chef-12.0.3/chef-12.0.3.gemspec 2017-01-10 10:47:47.173924839 -0500
+++ b/chef-12.0.3/chef-12.0.3.gemspec 2017-01-10 10:32:06.001903643 -0500
@@ -39,7 +39,7 @@
       s.add_runtime_dependency(%q<chef-zero>, ["~> 3.2"])
       s.add_runtime_dependency(%q<pry>, ["~> 0.9"])
       s.add_runtime_dependency(%q<plist>, ["~> 3.1.0"])
-      s.add_development_dependency(%q<rack>, [">= 0"])
+      s.add_development_dependency(%q<rack>, ["= 1.5.5"])
       s.add_development_dependency(%q<rake>, ["~> 10.1.0"])
       s.add_development_dependency(%q<rspec_junit_formatter>, ["~> 0.1.0"])
       s.add_development_dependency(%q<rspec-core>, ["~> 2.14.0"])
@@ -61,7 +61,7 @@
       s.add_dependency(%q<chef-zero>, ["~> 3.2"])
       s.add_dependency(%q<pry>, ["~> 0.9"])
       s.add_dependency(%q<plist>, ["~> 3.1.0"])
-      s.add_dependency(%q<rack>, [">= 0"])
+      s.add_dependency(%q<rack>, ["= 1.5.5"])
       s.add_dependency(%q<rake>, ["~> 10.1.0"])
       s.add_dependency(%q<rspec_junit_formatter>, ["~> 0.1.0"])
       s.add_dependency(%q<rspec-core>, ["~> 2.14.0"])
@@ -84,7 +84,7 @@
     s.add_dependency(%q<chef-zero>, ["~> 3.2"])
     s.add_dependency(%q<pry>, ["~> 0.9"])
     s.add_dependency(%q<plist>, ["~> 3.1.0"])
-    s.add_dependency(%q<rack>, [">= 0"])
+    s.add_dependency(%q<rack>, ["= 1.5.5"])
     s.add_dependency(%q<rake>, ["~> 10.1.0"])
     s.add_dependency(%q<rspec_junit_formatter>, ["~> 0.1.0"])
     s.add_dependency(%q<rspec-core>, ["~> 2.14.0"])
